bpmn-js-app/angular.json

```diff
@@ -13,6 +13,7 @@
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
+            "aot": true,
```

Comment:
What does this do? How does this contribute / improve to the example?

Reply:
@nikku: it is generated automatically by `ng update @angular/cli`